Clockify.Tests/Tests/TaskTests.cs

```diff
@@ -86,5 +86,69 @@ public async Task CreateTaskAsync_NullName_ShouldThrowArgumentException()
             await create.Should().ThrowAsync<ArgumentException>()
                 .WithMessage($"Value cannot be null. (Parameter '{nameof(TaskRequest.Name)}')");
         }
+
+        [Test]
+        public async Task UpdateTaskAsync_ShouldUpdateTask()
+        {
+            // Prepare some project
+            var projectRequest = new ProjectRequest { Name = "Update task project", Color = "#FF00FF" };
```

Comment:
Change:
`var projectRequest = new ProjectRequest { Name = "Update task project", Color = "#FF00FF" };`
  
To this 
`var projectRequest = new ProjectRequest { Name = "Update task project" + Guid.NewGuid()`, Color = "#FF00FF" };`